schema-registry/rest-service/src/test/java/com/hortonworks/registries/schemaregistry/avro/AvroSchemaRegistryClientTest.java

```diff
@@ -492,20 +493,23 @@ private void doTestSchemaVersionLifeCycleStates(SchemaValidationLevel validation
         SCHEMA_REGISTRY_CLIENT.enableSchemaVersion(schemaIdVersion_2.getSchemaVersionId());
         Assert.assertEquals(SchemaVersionLifecycleStates.ENABLED.getId(),
                             SCHEMA_REGISTRY_CLIENT.getSchemaVersionInfo(schemaIdVersion_2).getStateId());
-
-        SchemaIdVersion schemaIdVersion_4 =
-                SCHEMA_REGISTRY_CLIENT.addSchemaVersion(schemaName,
-                                                        new SchemaVersion(AvroSchemaRegistryClientUtil.getSchema("/schema-4.avsc"),
-                                                                          "Forth version of the schema, adds back name field, but different type",
-                                                                          SchemaVersionLifecycleStates.INITIATED.getId()));
+/*
```

Comment:
You can remove this and add the behavior change as part of the main PR description. You may need to update design doc also.

Reply:
Done